<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Basic Demo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MainWindow.qrc">
    <normaloff>:/MainDialog/AlliedVision.png</normaloff>:/MainDialog/AlliedVision.png</iconset>
  </property>
  <widget class="QWidget" name="m_pCentralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="m_pGraphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="m_pStatusBar"/>
  <widget class="QToolBar" name="m_pToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_pActionZoomIn"/>
   <addaction name="m_pActionZoomOut"/>
   <addaction name="m_pActionZoomReset"/>
   <addaction name="m_pActionFitScreen"/>
  </widget>
  <action name="m_pActionZoomIn">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainDialog/ZoomIn.png</normaloff>:/MainDialog/ZoomIn.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="m_pActionZoomOut">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainDialog/ZoomOut.png</normaloff>:/MainDialog/ZoomOut.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="m_pActionFitScreen">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainDialog/FitScreen.png</normaloff>:/MainDialog/FitScreen.png</iconset>
   </property>
   <property name="text">
    <string>Fit Screen</string>
   </property>
  </action>
  <action name="m_pActionZoomReset">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainDialog/ZoomReset.png</normaloff>:/MainDialog/ZoomReset.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Reset</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_pActionZoomIn</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ZoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pActionZoomOut</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ZoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pActionFitScreen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>FitScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pActionZoomReset</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ZoomReset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ZoomIn()</slot>
  <slot>ZoomOut()</slot>
  <slot>FitScreen()</slot>
  <slot>ZoomReset()</slot>
 </slots>
</ui>
